static-site/src/sections/individual-group-page.jsx

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import ScrollableAnchor, { configureAnchors } from "react-scrollable-anchor";
+import styled from 'styled-components';
+import {
+  HugeSpacer,
+  FlexCenter
+} from "../layouts/generalComponents";
+import * as splashStyles from "../components/splash/styles";
+import DatasetSelect from "../components/Datasets/dataset-select";
+import GenericPage from "../layouts/generic-page";
+import { fetchAndParseJSON } from "../util/datasetsHelpers";
+
+function Title({avatarSrc, children}) {
+  if (!children) return null;
+  const AvatarImg = styled.img`
+    width: 140px;
+    margin-right: 20px;
+    object-fit: contain;
+  `;
+  const TitleDiv = styled.div`
+    && {
+      font-weight: 500;
+      font-size: 26px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+  `;
+  return (
+    <div style={{display: "flex", justifyContent: "start", padding: "50px 0px 20px 0px"}}>
+      {avatarSrc ?
+        <AvatarImg alt="avatar" src={avatarSrc}/> :
+        null
+      }
+      <TitleDiv>
+        {children}
+      </TitleDiv>
+    </div>
+  );
+}
+
+function Byline({children}) {
+  if (!children) return null;
+  const Div = styled.div`
+    && {
+      font-size: 18px;
+      font-weight: 400;
+      line-height: 1.428;
+      color: #A9ADB1;
+    }
+  `;
+  return (<Div>{children}</Div>);
+}
+
+function Website({children}) {
+  if (!children) return null;
+  return (
+    <a href={children}
+      style={{color: "#A9ADB1", lineHeight: "1.0", textDecoration: "none", cursor: "pointer", fontWeight: "400", fontSize: "16px"}}
+    >
+      {children}
+    </a>
+  );
+}
+
+const GroupNotFound = (groupName) => (
+  <FlexCenter>
+    <splashStyles.CenteredFocusParagraph>
+      {`The Nextstrain Group "${groupName}" doesn't exist yet. `}
+      Please <a href="mailto:hello@nextstrain.org">contact us at hello@nextstrain.org </a>
+      if you believe this to be an error.</splashStyles.CenteredFocusParagraph>
+  </FlexCenter>
+);
+
+const OverviewContainer = styled.div`
+  text-align: justify;
+  font-size: 16px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  font-weight: 300;
+  color: var(--darkGrey);
+  line-height: 1.42857143;
+  margin: 0px auto 0px auto;
+  max-width: 900px;
+`;
+
+class Index extends React.Component {
+  constructor(props) {
+    super(props);
+    configureAnchors({ offset: -10 });
+    this.state = {
+      dataLoaded: false,
+      errorFetchingData: false,
+      groupNotFound: false,
+      groupName: undefined,
```

Comment:
Is there an advantage to having `groupName` in state, as opposed to `this.props.groupName`? (E.g. may we modify this variable in the future?)